src/ifconfig.interfaces.js

```diff
@@ -18,7 +19,7 @@ module.exports = function (cp) {
         return f(stderr);
       }
 
-      const routeCommand = options.gateway !== undefined && options.gateway.resolveHostNames ? 'route' : 'route -n';
+      const routeCommand = options.gateway !== undefined && options.gateway.resolveHostNames ? 'route' : 'route -n && route -6n';
```

Comment:
`route -6n` won't work everywhere (for instance it's not available on MacOSX).
However we could allow this command if it can be enabled through an option

Reply:
Added options.gateway.6n